### PR TITLE
DT-156 Rancher Desktop Validation for installing

### PR DESCRIPTION
**Pull request by DanielSotoEstrada**
Currently, in the Rancher Step we are testing if the plist file exist. If the file exists, the install step is never executed. However for some people Rancher Desktop is never updated and Kubernetes still showing enable.

We added additional validations to check if Rancher was manually installed and check if Kubernetes is enabled in order to continue with the install step.

### CODE REVIEW

File: lib/core/rancher/step.sh [hunk 1]
```diff
@@ -10,14 +10,31 @@ function ih::setup::core.rancher::help() {
...
+    fi
+
 
     # Check for PLIST FILE
     PLISTFILE="$HOME/Library/Preferences/io.rancherdesktop.profile.defaults.plist"
     if [ -f "$PLISTFILE" ]; then
-        return 0
+
+        # If this file exists, let's check kubernetes is not enabled
+        RESULT=$(rdctl list-settings | jq  '.kubernetes.enabled')
```
Comment (SteveRuble) [inline lib/core/rancher/step.sh:29]: I'm not sure what will happen if Rancher Desktop isn't running here, but this probably won't work.

File: lib/core/rancher/step.sh [hunk 2]
```diff
@@ -40,6 +57,8 @@ function ih::setup::core.rancher::install() {
     brew list ih-rancher > /dev/null 2>&1
     IHRANCHERINSTALLED=$?
 
+    echo " Rancher installed :" $RANCHERINSTALLED
+    echo " IH Rancher installed :" $RANCHERINSTALLED
```
Comment (SteveRuble) [inline lib/core/rancher/step.sh:61]: Did you mean `$IHRANCHERINSTALLED`?

Underscores make `ALLCAPS` `EASIER_TO_READ`.

You can (and probably should) write this as 
```
echo " IH Rancher installed : $RANCHERINSTALLED "
```

*BUT* why are you `echo`ing this at all?

Comment (DanielSotoEstrada) [reply lib/core/rancher/step.sh:61]: My mistake. I forgot to delete this when I was testing.

**Review by SteveRuble — APPROVED** [review]

File: lib/core/rancher/step.sh (hunk 1 above)
Comment (DanielSotoEstrada) [reply lib/core/rancher/step.sh]: I totally agree with you. I change the validation in order to see if our cask is listed or not. 
I let the file validation just like we had it.

**Review by alex-gomez-ih — APPROVED** [review]

**Pull request merged**